### PR TITLE
Add a troubleshooting section for the converter guidance

### PR DESCRIPTION
**Pull request by pshao25**
As titled.

### CODE REVIEW

File: docs/migrate-swagger/01-get-started.md [hunk 1]
```diff
@@ -73,6 +73,18 @@ This is the probably most critical step of the conversion. As you have pre-sorte
...
+
+```
+fatal   | The ResourceCollection RestorePointCollection (RequestPath: {Certain request path}) does not have a GetAll method
+```
+
+you could bypass it by the configuration in this [page](https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/readme.md#list-exception).
+
 ## How to Get Help
 
 - Ask questions in the [TypeSpec Discussions Channel](https://teams.microsoft.com/l/channel/19%3a906c1efbbec54dc8949ac736633e6bdf%40thread.skype/TypeSpec%2520Discussion%2520%25F0%259F%2590%25AE?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
```
Comment (lirenhe) [inline docs/migrate-swagger/01-get-started.md:90]: Can we use aka.ms link here?

File: docs/migrate-swagger/01-get-started.md [hunk 2]
```diff
@@ -73,6 +73,18 @@ This is the probably most critical step of the conversion. As you have pre-sorte
 
 - Review CI checks such as breaking changes and other failures.
 
+## Troubleshooting
+
+### Error "does not have a GetAll method"
+
+If converter fails withe error
```
Comment (lirenhe) [inline docs/migrate-swagger/01-get-started.md:80]: Typo?  'withe'

File: docs/migrate-swagger/01-get-started.md [hunk 3]
```diff
@@ -73,6 +73,18 @@ This is the probably most critical step of the conversion. As you have pre-sorte
...
+
+```
+fatal   | The ResourceCollection RestorePointCollection (RequestPath: {Certain request path}) does not have a GetAll method
+```
+
+you could bypass it by the configuration in this [page](https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/readme.md#list-exception).
```
Comment (lirenhe) [inline docs/migrate-swagger/01-get-started.md:86]: No quite sure about the action here.
By checking the link, do you suggest to update the config or update the Swagger to solve the problem?

File: docs/migrate-swagger/01-get-started.md (hunk 1 above)
Comment (pshao25) [reply docs/migrate-swagger/01-get-started.md:90]: We will delete this workaround after we remove the .net dependency. I prefer not to add an aka item.

Comment (ArthurMa1978) [reply docs/migrate-swagger/01-get-started.md:90]: Here is the short link for this channel https://aka.ms/typespec/discussions.
We need support TypeSpec migration for quite some time, so this line should remain in the document.

**Comment by pshao25** [comment]
Synced offline. We will add suppression by default. https://github.com/Azure/azure-sdk-tools/pull/8827

**Pull request closed without merging**